cts/scheduler/xml/bundle-order-fencing.xml

```diff
@@ -93,6 +94,12 @@
         </primitive>
       </bundle>
       <bundle id="galera-bundle">
+        <meta_attributes id="galera-meta_attributes">
+          <nvpair id="galera-meta_attributes-promoted-max" name="promoted-max" value="3"/>
```

Comment:
You added "-bundle" to the IDs for some resources and not others. Don't think it matters but correct me if I'm wrong.

Reply:
It doesn't matter, I was just trying to keep them consistent for anyone reading the XML, but not trying too hard ;)